src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -463,7 +463,7 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 				components.add(new Image("picture1", 20, 30, "picture1", true, new Procedure("condition1")));
 				components.add(new Image("picture2", 22, 31, "picture2", false, new Procedure("condition2")));
 				components.add(new Button("button1", 10, 10, "button1", 100, 200, new Procedure("procedure1")));
-				components.add(new Button("button2", 10, 10, "button2", 100, 200, null));
+				components.add(new Button("button2", 10, 10, "button2", 100, 200, null) );

```

Comment:
Unneeded change.

Reply:
Give me the time, please. I did it just after.